products/facilities/facdb/pipelines.py

```diff
@@ -364,19 +364,14 @@ def dsny_electronicsdrop(df: pd.DataFrame):
     return df
 
 
-def dycd_afterschoolprograms(df: pd.DataFrame):
-    df["address"] = df.location_1.apply(
-        lambda x: str(x).replace("nan", "").split("\n")[0][:-5]
-    )
-    df["zipcode"] = df.location_1.apply(
-        lambda x: str(x).replace("nan", "").split("\n")[0][-5:]
-    )
-    df["spatial"] = df.location_1.apply(
-        lambda x: str(x).replace("nan", "").split("\n")[-1]
-    )
-    df["spatial"] = df.spatial.apply(lambda x: x.replace("(", "").replace(")", ""))
-    df["longitude"] = df.spatial.apply(lambda x: x.split(",")[-1])
-    df["latitude"] = df.spatial.apply(lambda x: x.split(",")[0])
+def dycd_service_sites(df: pd.DataFrame):
+    def _nan_to_string(value: object) -> str:
+        return str(value).replace("nan", "")
+
+    df["address"] = df["street_address"].apply(_nan_to_string)
```

Comment:
Some specific odd cases accounted for before - zipcode last 5 characters of first line for example. Did you do any validation that the imported dataset was good to go?

Reply:
Ah again should have read a little more closely. Cool